packages/map/src/components/UsaMap/components/Territory/Territory.Hexagon.tsx

```diff
@@ -167,7 +180,6 @@ const TerritoryHexagon = ({
           style={{ fill: APP_FONT_COLOR }}
           alignmentBaseline='middle'
           transform={`translate(${centroid[0] + dx}, ${centroid[1] + dy})`}
-          onClick={handleShapeClick}
```

Comment:
I think I would leave these click events in. Even though it seems odd it can create dead clicks when users are click on the shape if they click directly on the labels/lines.